### PR TITLE
Added redfish_client package

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* Nightly

### CODE REVIEW

File: dependencies/buster/redfish_client/control [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Source: ruby-redfish-client
+Section: ruby
+Priority: optional
+Maintainer: Debian Ruby Extras Maintainers <pkg-ruby-extras-maintainers@lists.alioth.debian.org>
+Uploaders: Lukas Zapletal <lzap+pub@redhat.com>
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               git,
+               ruby-excon (>= 0.60),
+               ruby-server-sent-events (>= 0.1)
```
Comment (evgeni) [inline dependencies/buster/redfish_client/control:10]: I don't think those are build dependencies?

File: dependencies/buster/redfish_client/control [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Standards-Version: 4.3.0
+Vcs-Git: https://salsa.debian.org/ruby-team/ruby-redfish-client.git
+Vcs-Browser: https://salsa.debian.org/ruby-team/ruby-redfish-client
```
Comment (evgeni) [inline dependencies/buster/redfish_client/control:13]: This is not really true, so please drop :)

File: dependencies/buster/redfish_client/control [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Source: ruby-redfish-client
+Section: ruby
+Priority: optional
+Maintainer: Debian Ruby Extras Maintainers <pkg-ruby-extras-maintainers@lists.alioth.debian.org>
+Uploaders: Lukas Zapletal <lzap+pub@redhat.com>
```
Comment (evgeni) [inline dependencies/buster/redfish_client/control:5]:
```suggestion
Maintainer: Lukas Zapletal <lzap+pub@redhat.com>
```

Debian ruby won't touch this ;)

File: dependencies/buster/redfish_client/control (hunk 1 above)
Comment (lzap) [reply dependencies/buster/redfish_client/control:10]: Hmm weird `gem2deb` added these. Removing.

File: dependencies/buster/redfish_client/control (hunk 2 above)
Comment (lzap) [reply dependencies/buster/redfish_client/control:13]: Sure.

File: dependencies/buster/redfish_client/control (hunk 3 above)
Comment (lzap) [reply dependencies/buster/redfish_client/control:5]: Ok.

**Comment by lzap** [comment]
I guess I need to wait for the nightly repo to generate?

```
00:02:05.346  pbuilder-satisfydepends-dummy : Depends: ruby-server-sent-events (>= 0.1) which is a virtual package and is not provided by any available package
```

File: dependencies/buster/redfish_client/control (hunk 1 above)
Comment (evgeni) [reply dependencies/buster/redfish_client/control]: Hah. I spoke to quickly. Sorry. We run tests on debian, and that needs those, of course!

File: dependencies/buster/redfish_client/patches/decrease-excon-dependency.patch [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Pick any version of HTTP library.
+--- a/redfish_client.gemspec
++++ b/redfish_client.gemspec
+@@ -29,7 +29,7 @@
+ 
+   spec.required_ruby_version = "~> 2.1"
+ 
+-  spec.add_runtime_dependency "excon", "~> 0.71"
++  spec.add_runtime_dependency "excon"
```
Comment (evgeni) [inline dependencies/buster/redfish_client/patches/decrease-excon-dependency.patch:9]: why?

Comment (lzap) [reply dependencies/buster/redfish_client/patches/decrease-excon-dependency.patch:9]: This gem is in Debian in old version, it's a HTTP client library very unlikely it really needs the most recent one.

Comment (evgeni) [reply dependencies/buster/redfish_client/patches/decrease-excon-dependency.patch:9]: Ah, okay, we don't build it ourself. Well, then.

**Comment by evgeni** [comment]
updated version in #6061

**Comment by lzap** [comment]
This was merged as https://github.com/theforeman/foreman-packaging/pull/6061

**Pull request closed without merging**